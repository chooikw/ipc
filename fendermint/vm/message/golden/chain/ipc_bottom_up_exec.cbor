a163497063a16c426f74746f6d557045786563a2676d657373616765a4697375626e65745f6964a264726f6f7401686368696c6472656e8256040a410176d465b630ecda443cc7967111c9aae5286356040a44235d0a9e1501a43211d9fe787f69d3198e3cbd666865696768741a4242c88d756e6578745f76616c696461746f725f7365745f69641b3cce6c1b2e9e71e672626f74746f6d5f75705f6d65737361676573d82a582400015dc8bde8781c407245098aea7ff74753c0105fa1064d5ca592eb08e7828bb57291f76b6365727469666963617465a16a7369676e61747572657383a26976616c696461746f7255010163eb7e5b517c152fed42af36517cadf4e419a7697369676e617475726546018afdb22954a26976616c696461746f7255017dfee5375432606893ff8cff8dc800fbe2b4168b697369676e6174757265460263eb72018da26976616c696461746f72583103a1ed6e95da98564c011cf2f346eb5fa184aa250fa0c29aff4899e37686c0517579deea6801f6ef3af3b987a3e95a00ef697369676e617475726542021d