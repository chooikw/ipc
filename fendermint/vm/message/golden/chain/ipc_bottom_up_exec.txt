Ipc(BottomUpExec(CertifiedMessage { message: BottomUpCheckpoint { subnet_id: SubnetID { root: 1, children: [Address("f410fieaxnvdfwyyozwsehtdzm4irzgvokkddjrywboa"), Address("f410fiqrv2cu6cua2imqr3h7hq73j2mmy4pf5453srhy")] }, height: 1111672973, next_validator_set_id: 4381558351480123878, bottom_up_messages: Cid(bafo4rppipaoea4sfbgfou77xi5j4aec7uede2xffslvqrz4cro2xfepx) }, certificate: MultiSig { signatures: [ValidatorSignature { validator: Address("f1afr6w7s3kf6bkl7nikxtmul4vx2oignhlo7dj2a"), signature: Signature { sig_type: Secp256k1, bytes: [138, 253, 178, 41, 84] } }, ValidatorSignature { validator: Address("f1px7okn2ugjqgre77rt7y3saa7prlifula3ibxnq"), signature: Signature { sig_type: BLS, bytes: [99, 235, 114, 1, 141] } }, ValidatorSignature { validator: Address("f3uhww5fo2tbleyai46lzun227ugckujipudbjv72ithrxnbwakf2xtxxknaa7n3z26o4ypi7jliao66s6i5fa"), signature: Signature { sig_type: BLS, bytes: [29] } }] } }))